.github/workflows/pullrequests.yml

```diff
@@ -0,0 +1,59 @@
+name: flask-rebar Pull Request Tests
+
+on:
+- pull_request
+
+jobs:
+  tests:
+    name: Testing on Python ${{ matrix.python-version }} and ${{ matrix.os }} 
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 20
+      fail-fast: false
+      matrix:
+        python:
+        - 2.7
+        - 3.5
+        - 3.6
+        - 3.7
+        - 3.8
+        libraries:
+        - marshmallow=='2.13.*' flask=='0.10.*'
+        - marshmallow=='2.14.*' flask=='0.11.*'
+        - marshmallow=='2.15.*' flask=='0.12.*'
```

Comment:
I think we should drop support for flask<1.0 in v2 @RookieRick @twosigmajab

Reply:
Seems reasonable - I'm tempted to say we do a "soft drop".. i.e., we could leave `"Flask>=0.10,<2"` in setup.py `install_requires` and just remove the explicit test steps here against pre 1.0.  I don't know if there are any (sane) people in the world using pre-1.0 versions of Flask **_and_** using Flask-Rebar, but if they exist, I don't want to break their stuff unless we really have to (i.e., if we made some change that relied on a Flask 1.0+ feature making us no longer compatible with Flask 0.* we could update setup.py as well - and per our discussion my #149 we could justify  doing so with just a minor version bump ;) )